src/task.ts

```diff
@@ -388,19 +372,13 @@ export const requeueUnterminatedTasks = async (ctx: Context, bot: Probot): Promi
   }
 };
 
-export const getSendTaskMatrixResult =
-  (matrix: MatrixClient, task: ApiTask) =>
-  async (message: CommandOutput): Promise<void> => {
-    try {
-      await matrix.sendText(
-        task.matrixRoom,
-        `Task ID ${task.id} has finished with message "${message instanceof Error ? displayError(message) : message}"`,
-      );
-    } catch (rawError) {
-      const error = intoError(rawError);
-      log.error(extractRequestError(error), "Caught error when sending Matrix message");
-    }
-  };
+export const apiTaskResult =
+  (ctx: Context) =>
+  async (message: CommandOutput): Promise<void> =>
+    await new Promise((resolve) => {
+      ctx.logger.info({}, message);
+      resolve();
+    });
```

Comment:
any reason for the `new Promise` thing here?

Reply:
just following the contract, as the callback expects promise there

Reply:
in a meanwhile we're just logging this, later when we have a ui, i guess this will go somewhere to DB records like history of runs

Reply:
Well, I guess simplification is an ongoing progress :)